test/src/test/java/hudson/AboutJenkinsTest.java

```diff
@@ -45,75 +45,101 @@ class AboutJenkinsTest {
+            webClient.withThrowExceptionOnFailingStatusCode(false);
+            return webClient.goTo(ABOUT_PAGE_URL);
         }
+    }
 
-        { // manager-readonly can access it
-            wc.login(MANAGER_READONLY);
-            HtmlPage page = wc.goTo("about/");
-            assertEquals(HttpURLConnection.HTTP_OK, page.getWebResponse().getStatusCode());
-        }
+    @Test
+    @Issue("SECURITY-771")
+    void usersWithElevatedPermissionsCanSeeAboutPage() throws Exception {
+        // ADMINISTER permission: admin can see About Jenkins page
+        HtmlPage adminPage = accessAsUser(ADMIN_USER);
+        assertEquals(HttpURLConnection.HTTP_OK, adminPage.getWebResponse().getStatusCode());
+        assertThat(adminPage.getWebResponse().getContentAsString(), containsString(MAVENIZED_DEPS_TEXT));
+        assertThat(adminPage.getTitleText(), containsString(ABOUT_PAGE_TITLE));
+
+        // MANAGE permission: manager can see About Jenkins page
+        HtmlPage managerPage = accessAsUser(MANAGER_USER);
+        assertEquals(HttpURLConnection.HTTP_OK, managerPage.getWebResponse().getStatusCode());
+        assertThat(managerPage.getWebResponse().getContentAsString(), containsString(MAVENIZED_DEPS_TEXT));
+        assertThat(managerPage.getTitleText(), containsString(ABOUT_PAGE_TITLE));
+
+        // MANAGE + SYSTEM_READ permissions: manager-readonly can see About Jenkins page
+        HtmlPage managerReadonlyPage = accessAsUser(MANAGER_READONLY_USER);
+        assertEquals(HttpURLConnection.HTTP_OK, managerReadonlyPage.getWebResponse().getStatusCode());
+        assertThat(managerReadonlyPage.getWebResponse().getContentAsString(), containsString(MAVENIZED_DEPS_TEXT));
+        assertThat(managerReadonlyPage.getTitleText(), containsString(ABOUT_PAGE_TITLE));
     }
 
-}
+    @Test
+    @Issue("SECURITY-771")
+    void usersWithBasicPermissionsCannotSeeAboutPage() throws Exception {
+        // anonymous user cannot see About Jenkins page -> redirect to sign in page
+        HtmlPage anonymousPage = accessAsAnonymous();
+        assertEquals(HttpURLConnection.HTTP_OK, anonymousPage.getWebResponse().getStatusCode());
+        assertThat(anonymousPage.getTitleText(), containsString(SIGN_IN_PAGE_TITLE));
+
+        // only READ permission: user cannot see About Jenkins page -> redirect to Access Denied Jenkins page
+        HtmlPage userPage = accessAsUser(REGULAR_USER);
+        assertEquals(HttpURLConnection.HTTP_FORBIDDEN, userPage.getWebResponse().getStatusCode());
+        assertThat(userPage.getTitleText(), containsString(JENKINS_PAGE_TITLE));
+
+        // SYSTEM_READ permission: readonly cannot see About Jenkins page -> redirect to Access Denied Jenkins page
+        HtmlPage readonlyPage = accessAsUser(READONLY_USER);
+        assertEquals(HttpURLConnection.HTTP_FORBIDDEN, readonlyPage.getWebResponse().getStatusCode());
+        assertThat(readonlyPage.getTitleText(), containsString(JENKINS_PAGE_TITLE));
```

Comment:
This is wrong. Overall/SystemRead is a powerful permission and should be able to read the page, but you never grant this user Overall/Read (unlike the user with Manage).